generators/server/prompts.js

```diff
@@ -31,14 +31,42 @@ function askForServerSideOpts() {
                 /^([a-z_A-Z]\w+(?:\.[a-z_A-Z]\w+)*)$/.test(input) ? true : 'The namespace you have provided is not a valid C# namespace',
             message: 'What is your default C# namespace?',
             default: 'MyCompany'
+        },
+        {
+            type: 'list',
+            name: 'database',
+            message: 'Wchich database do you want to use',
+            choices: [
+                {
+                    value: 'sqllite',
+                    name: 'sql lite in memory'
```

Comment:
We should respect the product namin/casing
sql lite => SQLite (cf. https://www.sqlite.org/index.html)